templates/web/guides/models/pins.md.erb

```diff
@@ -222,6 +222,39 @@ ground_pin_groups.each do |id, group|
 end
 ~~~
   
+#### Virtual Pin Modelling
+
+The API supports modelling virtual pins.  Virtual pins are essentially user-defined pins that may not necessarily be
+associated with the SoC, such as:
+  *  Tester utility pins that control relay states on the device interface board (DIB)
+  *  Tester channels to control components on the DIB
+
+~~~ruby
+add_virtual_pin(:virtual1, type: :utility_pin)
+add_virtual_pin(:virtual2, type: :utility_pin)
+add_virtual_pin(:virtual3, type: :relay_pin)
+
+add_virtual_pin_group :utility, :virtual1, :virtual2
+add_virtual_pin_group :relay, :relay_pin
```

Comment:
It certainly should.  Thanks for catching this.

--Aaron

From: Stephen McGinty [mailto:notifications@github.com]
Sent: Thursday, March 10, 2016 4:42 AM
To: Origen-SDK/origen <origen@noreply.github.com>
Cc: Aaron Burgmeier <aaron.burgmeier@nxp.com>
Subject: Re: [origen] Updates to Virtual pin model (#35)


In templates/web/guides/models/pins.md.erb<https://github.com/Origen-SDK/origen/pull/35#discussion_r55661578>:

> @@ -222,6 +222,39 @@ ground_pin_groups.each do |id, group|

>  end

>  ~~~

>

> +#### Virtual Pin Modelling

> +

> +The API supports modelling virtual pins.  Virtual pins are essentially user-defined pins that may not necessarily be

> +associated with the SoC, such as:

> +  *  Tester utility pins that control relay states on the device interface board (DIB)

> +  *  Tester channels to control components on the DIB

> +

> +~~~ruby

> +add_virtual_pin(:virtual1, type: :utility_pin)

> +add_virtual_pin(:virtual2, type: :utility_pin)

> +add_virtual_pin(:virtual3, type: :relay_pin)

> +

> +add_virtual_pin_group :utility, :virtual1, :virtual2

> +add_virtual_pin_group :relay, :relay_pin

I think this should say virtual3?

—
Reply to this email directly or view it on GitHub<https://github.com/Origen-SDK/origen/pull/35/files#r55661578>.